experimental/cloudstrapper/playbooks/roles/vars/build.yaml

```diff
@@ -17,12 +17,13 @@ buildGwInstanceType: t2.medium
 # Ubuntu 20.04 in us-east-1: ami-0885b1f6bd170450c
 # Ubuntu 20.04 in ap-northeast-1: ami-0f2dd5fc989207c82
 # Ubuntu 20.04 in ca-central-1: ami-02e44367276fe7adc
-buildUbuntuAmi: ami-0f2dd5fc989207c82
+# created by Art Lee Ubuntu 20.04 in us-west-1:ami-0b30c5a1b42eb8d11
+buildUbuntuAmi: ami-0b30c5a1b42eb8d11
 
 #Debian 4.9.0.9 AMI n us-east-2:  ami-0c305b7817e9fd559
 #Debian 4.9.0.9 AMI n ap-northeast-1: ami-0aac5cd08e351ff0f
-buildDebianAmi: ami-0aac5cd08e351ff0f
-buildAgwAmiName: testImgAgw14
+buildDebianAmi: ami-0e7b52505c766bf98
```

Comment:
can all those be dynamic?

Reply:
there is a duplicate but these are legacy now
the short answer is not at this time